### ISSUE TITLE
Use new Task API to start/stop (and possibly monitor) tasks

### ISSUE THREAD

**Issue by g-arjones** [rock-core/vscode-rock#56]
This would be useful to start/stop/monitor syskit rest server and autoproj watch process. Reference: https://github.com/Microsoft/vscode/commit/2f5061ff2d68637c1ab9c10d06a188333f29c22d

**Comment by doudou** [comment]
Oh that is good news. It felt so wrong not being able to check that things were running properly.

**Comment by g-arjones** [comment]
Yes.. Sounds great. We will also get dedicated output channels for free..
